claat/render/md.go

```diff
@@ -272,6 +272,22 @@ func (mw *mdWriter) infobox(n *types.InfoboxNode) {
 	mw.Prefix = ""
 }
 
+func (mw *mdWriter) survey(n *types.SurveyNode) {
+	mw.newBlock()
+	mw.writeString("<form>")
+	mw.writeBytes(newLine)
+	for _, g := range n.Groups {
+		// TODO: do these values need to be escaped like in the html renderer?
```

Comment:
Have you tested this?

Reply:
I've updated the PR to escape the html. I'll share some internal testing links with you in chat